migrations/20201003211651_make-name-null.sql

```diff
@@ -0,0 +1,5 @@
+-- Add migration script here
+ALTER TABLE users
+ALTER COLUMN name DROP NOT NULL;
+ALTER TABLE users
+ALTER COLUMN name DROP DEFAULT;
```

Comment:
I already removed the default in my bug-fix PR, it may cause issues if done twice.

Reply:
It worked fine for me